bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/HBaseImportOptions.java

```diff
@@ -0,0 +1,23 @@
+package com.google.cloud.bigtable.dataflow;
+
+import com.google.cloud.dataflow.sdk.options.Default;
+import com.google.cloud.dataflow.sdk.options.Description;
+
+/**
+ * An extension of {@link CloudBigtableOptions} that contains additional configuration
+ * for importing HBase sequence files into Cloud Bigtable.
+ */
+public interface HBaseImportOptions extends CloudBigtableOptions {
+  @Description("Prefix or name of the input file(s).")
+  String getFilePattern();
+
+  void setFilePattern(String filePattern);
+
+  @Description(
```

Comment:
I think this would be a one-off change. 
In HBase 0.98 serialization of Result changed from an ad hoc way to using protocol buffer utils. The sequence file format did not change. Proto should allow HBase to hide future changes from api.